### PR TITLE
Rename some replication metrics and fix 3-node dev configs

### PR DESCRIPTION
**Pull request by xiahome**
1. Make the metrics name for getRequestError, localStoreError, and metadataRequestError to include partitionId instead of the path of corresponding replica.
2. Fix configs for the 3-node dev setup, in particular fixes server ports and removes ssl ports.

Test: 1.  ./gradlew build && ./gradlew test
         2. visually verified metrics in JConsole in a local three-node setup.

### CODE REVIEW

File: config/HardwareLayoutLocalThreeNodeOneDisk.json [hunk 1]
```diff
@@ -14,8 +14,7 @@
                     ],
                     "hardwareState": "AVAILABLE",
                     "hostname": "localhost",
-                    "port": 6670,
-                    "sslport": 7670
```
Comment (cgtz) [inline config/HardwareLayoutLocalThreeNodeOneDisk.json]: could you keep the version with SSL ports around too?

**Comment by cgtz** [comment]
mostly LGTM, I just have 1 comment.

**Comment by xiahome** [comment]
@cgtz addressed your comments.

File: config/PartitionLayoutLocalOnePartitionThreeReplicasWithSsl.json [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+        {
+            "id": 0,
+            "partitionState": "READ_WRITE",
+            "replicaCapacityInBytes": 10737418240,
+            "replicas": [
+                {
+                    "hostname": "localhost",
+                    "mountPath": "/tmp/1",
+                    "port": 6670,
+                    "sslport": 7670
```
Comment (vgkholla) [inline config/PartitionLayoutLocalOnePartitionThreeReplicasWithSsl.json]: I don't think the partition layout has ssl ports. It is the hardware layout. Look at HardwareLayout.json and HardwareLayoutSsl.json (notice that there is only PartitionLayout.json and no PartitionLayoutSsl.json). @nsivabalan should be able to confirm.

**Comment by vgkholla** [comment]
LGTM except for one comment.

File: config/PartitionLayoutLocalOnePartitionThreeReplicasWithSsl.json (hunk 2 above)
Comment (nsivabalan) [inline config/PartitionLayoutLocalOnePartitionThreeReplicasWithSsl.json]: Yes, PartitionLayout doesn't have port nos.

File: config/server1.properties [hunk 3]
```diff
@@ -10,4 +10,4 @@
 # CONDITIONS OF ANY KIND, either express or implied.
 #
 host.name=localhost
-port=6667
+port=6670
```
Comment (nsivabalan) [inline config/server1.properties]: If you don't mind, can you also create 3 more files for ssl. (serverSsl1.properties, serverSsl2.properties, etc) for ssl purposes

**Comment by nsivabalan** [comment]
done with my comments

**Comment by xiahome** [comment]
@cgtz @nsivabalan @vgkholla Addressed comment. PTAL

**Comment by vgkholla** [comment]
LGTM

**Comment by nsivabalan** [comment]
LGTM

**Pull request merged**